docker-compose.yml

```diff
@@ -130,6 +130,27 @@ services:
       INFLUXDB_INIT_PWD: "${TTN_DASHBOARD_INFLUXDB_ADMIN_PASSWORD:-!notset}"
       PRE_CREATE_DB: "${TTN_DASHBOARD_INFLUXDB_INITIAL_DATABASE_NAME:-demo}"
 
+  ############# Murugan Changes for influxdb backup
+
+  influxdb-backup:
+    restart: unless-stopped
+    image: muruganchandrasekar/influxdb_backup_1.2.4:latest
```

Comment:
please do not use a private image with your name -- this is going to be shared by many people. Please use the public version (just as the other uses do).

Reply:
I've created Docker file for influx backup and then built image using the docker file. So I pushed to my dockerhub repository. How to push it to be public?.